app/src/main/res/layout/fragment_global.xml

```diff
@@ -0,0 +1,225 @@
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/title" />
+
+
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/materialCardView"
+            android:layout_width="@dimen/cards_width"
+            android:layout_height="@dimen/cards_height"
+            android:layout_marginTop="@dimen/cards_margin_parent_top"
+            android:layout_marginEnd="@dimen/cards_margin_parent_end"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <TextView
+                    android:id="@+id/total_cases_title"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/text_card_padding"
+                    android:text="@string/total_cases_title"
+                    android:textColor="?android:attr/textColorPrimary"
+                    android:textSize="@dimen/home_text_size"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <TextView
+                    android:id="@+id/total_cases"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAlignment="center"
+                    android:textSize="@dimen/cases_numbers_size"
+                    app:layout_constraintBottom_toTopOf="@+id/percentageCasesDifference"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    tools:text="1244352" />
+
+                <TextView
+                    android:id="@+id/percentageCasesDifference"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textSize="@dimen/increase_percentage_text_size"
+                    android:layout_marginBottom="@dimen/text_card_padding"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    tools:text="@string/percentage_example" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+        </com.google.android.material.card.MaterialCardView>
+
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/materialCardView2"
```

Comment:
in this view you have the same view structure 3 times, 

`MeterialCardView`
 -> `ConstraintLayout`
  -> `TextView`
  -> `TextView`
  -> `TextView`

as a good experiment, you could create your own custom view which contains this structure and that it receives the data that it needs to build and update itself.